client/extensions/belgaCoverage/src/belga-coverage-carousel.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+
+import {ISuperdesk} from 'superdesk-api';
+import {Badge, Carousel, IconButton} from 'superdesk-ui-framework';
+import {IBelgaCoverage, getCoverageInfo, getCoverageImages, IBelgaImage} from './belga-image-api';
+import 'superdesk-ui-framework/dist/superdesk-ui.bundle.css';
+
+interface IProps {
+    preview?: boolean;
+    coverageId: string;
+    removeCoverage?: () => void;
+    superdesk: ISuperdesk;
+}
+
+interface IState {
+    loading: boolean;
+    coverage: IBelgaCoverage | null;
+    images: Array<IBelgaImage> | null;
+}
+
+export default class BelgaCoverageCarousel extends React.PureComponent<IProps, IState> {
+    constructor(props: IProps) {
+        super(props);
+
+        this.state = {
+            loading: true,
+            coverage: null,
+            images: null,
+        };
+    }
+
+    componentDidMount() {
+        Promise.all([
+            getCoverageInfo(this.props.coverageId),
+            getCoverageImages(this.props.coverageId, 8),
+        ]).then(([coverage, images]) => {
+            this.setState({
+                loading: false,
+                coverage,
+                images,
+            });
+        }).catch((err) => {
+            console.error(err);
+            this.setState({
+                loading: false,
+            });
+        });
+    }
+
+    render() {
+        if (this.state.loading) {
+            return (
+                <div className="sd-loader" />
+            );
+        }
+
+        const {Alert} = this.props.superdesk.components;
```

Comment:
If you are targeting `develop` branch of the client, it's now possible to import `superdesk` in [the root scope of the file](https://github.com/tomaskikutis/superdesk-planning/blob/assignments-widget-in-the-toolbar/client/planning-extension/src/assignments-overview/assignments-overview-list-item.tsx#L7) now